classes/issue/Collector.php

```diff
@@ -348,8 +348,8 @@ public function getQueryBuilder(): Builder
         // Context
         // Never permit a query without a context_id unless the PKPApplication::CONTEXT_ID_ALL wildcard
         // has been explicitly set
-        if (!in_array(PKPApplication::CONTEXT_ID_ALL, $this->contextIds)) {
-            $q->whereIn('i.journal_id', $this->contextIds);
+        if (!in_array(PKPApplication::CONTEXT_ID_ALL, $this->contextIds ?? [])) {
```

Comment:
At this point `$this->contextIds` might be `null`.

Reply:
I'll probably update to follow the existing pattern:
https://github.com/pkp/pkp-lib/blob/53736488b005a72a3485ed012da389a07cda54fb/classes/submission/Collector.php#L267-L271

Reply:
Doesn't seem related to the events but linked example looks like a good way to proceed 👍